Find a functionally equivalent code:android.widget.FastScroller.isPointInside:COMMENT
Method Modifier: private     
Comment:/**
 * Returns whether a coordinate is inside the scroller's activation area. If
 * there is a track image, touching anywhere within the thumb-width of the
 * track activates scrolling. Otherwise, the user has to touch inside thumb
 * itself.
 *
 * @param x The x-coordinate.
 * @param y The y-coordinate.
 * @return Whether the coordinate is inside the scroller's activation area.
 */

@@ -1,3 +1,3 @@
 {
-    return isPointInsideX(x) && (mHasTrackImage || isPointInsideY(y));
+    return isPointInsideX(x) && (mTrackDrawable != null || isPointInsideY(y));
 }

